js/less/bqplot.less

```diff
@@ -63,7 +63,7 @@
   --bq-pie-polyline-stroke: var(--bq-content-font-color);
 
   --bq-network-node-element-fill: #666;
-  --bq-linkarrow-fill: #ccc;
+  --bq-linkarrow-fill: rgb(0, 0, 0);
```

Comment:
I guess you want to use a CSS variable for that?

Reply:
Yes, absolutely 😄